bundles/EcommerceFrameworkBundle/Tools/Installer.php

```diff
@@ -414,4 +409,12 @@ public function needsReloadAfterInstall()
     {
         return true;
     }
+
+    /**
+     * @return Schema
+     */
+    protected function getSchema(): Schema
+    {
+        return $this->schema ??= $this->db->getSchemaManager()->createSchema();
```

Comment:
I think we need to do it the same way as we did it previously ... since we cannot add the `getSchemaManager` method to the interface. 

So do something like that: 
```
        if ($this->db instanceof Connection) {
            $this->schema ??= $this->db->getSchemaManager()->createSchema();
        } else {
            throw new \Exception('Cannot create schema');
        }
```

Reply:
Implemented as suggested.